core/templates/dev/head/pages/exploration-editor-page/translation-tab/translation-tab.directive.ts

```diff
@@ -62,11 +65,13 @@ angular.module('oppia').directive('translationTab', ['UrlInterpolationService',
         'StateEditorService', 'StateRecordedVoiceoversService',
         'StateTutorialFirstTimeService', 'StateWrittenTranslationsService',
         'TranslationTabActiveModeService',
+        'UserExplorationPermissionsService',
```

Comment:
Can we move this to the line above?

Reply:
Doing that would mean we will need to move line 74 to line 73, and that will cause some lint issues because line 73 will become 82 characters long